easybuild/easyconfigs/c/cooltools/cooltools-0.7.1-foss-2024a.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'PythonBundle'
+
+name = 'cooltools'
+version = '0.7.1'
+
+homepage = 'cooltools.readthedocs.io/en/latest/index.html'
+description = """cooltools provides a suite of computational tools with a paired python API and
+ command line access, which facilitates workflows either on high-performance
+ computing clusters or via custom analysis notebooks. As part of the Open2C
+ ecosystem, cooltools also provides detailed introductions to key concepts in
+ Hi-C-data analysis with interactive notebook documentation."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Cython', '3.0.10'),
```

Comment:
this should be a build dependency from 2024a on

Reply:
At least Python is also a runtime dependency. Should I list it twice?